coalib/logging_config.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "handlers": {
+        "colored": {
+            "class": "logging.StreamHandler",
+            "formatter": "raw",
+            "stream": "ext://sys.stdout"
+        },
+        "raw": {
+            "class": "logging.StreamHandler",
+            "formatter": "raw",
+            "stream": "ext://sys.stdout"
+        }
```

Comment:
hm those two handlers look pretty similar to each other... the same. How does the colored one get the color?